movies/2005/walk-the-line.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Walk the Line",
+    "year": 2005,
+    "runtime": 136,
+    "categories": [
+        "biography",
+        "drama",
+        "music"

```

Comment:
Please remove "music" from the list 😄

Reply:
in the contribution file we don't have rules about categories... except about animation and sci-fi [https://github.com/mikeleguedes/json-movie-list/blob/master/contributing.markdown](url)

Reply:
Since there is not a rule about it do I need to change it or is it fine to stay there. It is one of the categories listed by IMDb - https://www.imdb.com/title/tt0358273/?ref_=adv_li_tt

Reply:
@mikeleguedes @leonlafa Please see my above comment.

Reply:
> @mikeleguedes @leonlafa Please see my above comment.

I think it's ok to stay